meal_master_fullstack/Models/ShoppingList.cs

```diff
@@ -0,0 +1,13 @@
+public class ShoppingList
+{
+    public int Id { get; set; }
+    public string Name { get; set; }
+    public string Quantity { get; set; }
```

Comment:
The list won't have a quantity, the items on the list will. We should have an array of objects (if it were javascript). Instead of name, quantity, and isChecked, we should have a list of items on the list that each have those properties.

Reply:
That is a good point. I think if we just rename this to "ShoppingListItems" That should get the point across? The only GET function is to get all of the shopping list items by the account, so I think it is just a naming convention thing?

Reply:
Yeah, that works

Reply:
I have changed the names, but now I am having trouble spinning up the project.  I don't think I'll have any more time today to work on this, but it should work, but something with spinning up the project is now not working right.